build/dependencies.props

```diff
@@ -36,6 +36,8 @@
     <MoqPackageVersion>4.10.0</MoqPackageVersion>
     <NETStandardLibrary20PackageVersion>2.0.3</NETStandardLibrary20PackageVersion>
     <NewtonsoftJsonPackageVersion>11.0.2</NewtonsoftJsonPackageVersion>
+    <SeleniumWebDriverChromeDriverPackageVersion>2.43.0</SeleniumWebDriverChromeDriverPackageVersion>
```

Comment:
cc @Eilon, these are new dependencies on third party software. Can we get these through OSS?

Reply:
Good to go.

Reply:
This can now be down non-atomically, apparently.